next-i18next.config.js

```diff
@@ -1,6 +1,12 @@
+// @ts-check
+
+/**
+ * @type {import('next-i18next').UserConfig}
```

Comment:
Is this commented out, or is it doing something; would like to understand what it's doing if it is.